docs/index.md

```diff
@@ -98,6 +98,8 @@ releases:
     value: false
 ```
 
+**NOTE:** The `helm-diff` plugin is required for helmfile to apply changes. Please ensure the [Helm Diff Plugin](https://github.com/databus23/helm-diff#readme) is installed.
```

Comment:
you can install it by running `helmfile init`. @rlnchow

Reply:
I will update the doc. Thank you @yxxhero